native_words.asm

```diff
@@ -2297,22 +2297,36 @@ _specials_done:
                 adc cp+1
                 sta cp+1
 
-                bra _done
+                rts
```

Comment:
Just a note on RTS in the middle of a word - you can get away with it here because `COMPILE,` is flagged as a NeverNative word, but most words need to have a single RTS at the very end to allow them to be natively compiled.  That's why you'll see things like `bra _done` all over the place.